packages/dappmanager/src/modules/migrations/addAliasToRunningContainers.ts

```diff
@@ -26,140 +26,176 @@ const dncoreNetworkName = params.DNP_PRIVATE_NETWORK_NAME;
+    migrateCoreNetworkAndAliasInCompose(container, aliases);
+
+    // Adds aliases to the container network
+    for (const alias of aliases) {
+      const currentEndpointConfig = await getDnCoreNetworkContainerConfig(container.containerName);
+      if (!hasAlias(currentEndpointConfig, alias)) {
+        const updatedConfig = updateEndpointConfig(currentEndpointConfig, alias);
+        await updateContainerNetwork(dncoreNetworkName, container, updatedConfig);
+        logs.info(`alias ${alias} added to ${container.containerName}`);
+      }
+    }
+  }
 }
 
-/**
- * Get compose file network and compose network settings from dncore_network
- * And rewrites the compose with the core network edited
+/** Gets the docker-compose.yml file of the given `container` and adds one or more alias 
+ * to the service that started `container`. All alias are added to the network defined by
+ * `params.DNP_PRIVATE_NETWORK_NAME`.
+ * 
+ * @param container PackageContainer
+ * @param aliases string[]
+ * @returns void
  */
 export function migrateCoreNetworkAndAliasInCompose(
   container: PackageContainer,
-  alias: string
+  aliases: string[]
 ): void {
   const compose = new ComposeFileEditor(container.dnpName, container.isCore);
 
-  // 1. Get compose network settings
-  const composeNetwork = compose.getComposeNetwork(
-    params.DNP_PRIVATE_NETWORK_NAME
-  );
-
-  // 2. Get compose service network settings
-  const composeService = compose.services()[container.serviceName];
-
+  // Gets all the networks defined in the service
   const serviceNetworks = parseServiceNetworks(
-    composeService.get().networks || {}
+    compose.services()[container.serviceName].get().networks || {}
   );
+  
+  // Gets current aliases of "params.DNP_PRIVATE_NETWORK_NAME", usually dncore_network
+  const currentAliases = serviceNetworks[params.DNP_PRIVATE_NETWORK_NAME]?.aliases || [];
 
-  const serviceNetwork =
-    serviceNetworks[params.DNP_PRIVATE_NETWORK_NAME_FROM_CORE] ?? null;
+  //add new aliases to current aliases set
+  const newAliases = uniq([...currentAliases, ...aliases]);
 
-  // 3. Check if migration was done
-  if (
-    isComposeNetworkAndAliasMigrated(
-      composeNetwork,
-      serviceNetwork,
-      compose.compose.version,
-      alias
-    )
-  )
-    return;
+  // Gets the network "dncore_network" from the general compose file 
+  const composeNetwork = compose.getComposeNetwork(params.DNP_PRIVATE_NETWORK_NAME);
 
-  // 4. Ensure compose file version 3.5
+  // Gets the old network "network" from the service compose file. This is an old network that is to be removed".
+  // If it is already removed, return null
+  const serviceNetwork = serviceNetworks[params.DNP_PRIVATE_NETWORK_NAME_FROM_CORE] ?? null;
+
+  // Return if migration was done, compose is already updated
+  if (isComposeNetworkAndAliasMigrated(composeNetwork, serviceNetwork, compose.compose.version, newAliases)) return
+  
+  // Ensure/update compose file version 3.5
```

Comment:
Are we sure this is the only format that we allow? Can someone install a package with a different format?